PosProject/src/main/java/com/pos/posproject/ejb/ProductCatalogBean.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.pos.posproject.ejb;
+
+import com.pos.posproject.common.ProductCatalogDetails;
+import com.pos.posproject.entity.ProductCatalog;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Logger;
+import javax.ejb.EJBException;
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+/**
+ *
+ * @author Monica
+ */
+@Stateless
+public class ProductCatalogBean {
+
```

Comment:
We should do a method here where we that do the mapping from the collection of product to a collection of product details